src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -310,6 +311,38 @@ def _validate_metadata_names(self):
         if names:
             helpers.validate_metadata_names(self.dataset, names)
 
+    def find_similar(
+        self,
+        vector_name: str,
+        value: Optional[List[float]],
+        record: Optional[RemoteFeedbackRecord],
+        max_results: int = 50,
+    ):
+        try:
```

Comment:
We should add a check to see whether at least one of `value` or `record` is provided and raise a `ValueError` if both are provided or if any is provided

Reply:
We're doing that on the `root_validator` of `FeedbackRecordsSearchVectorQuery`

Reply:
Yes I know, but do we want to keep that logic there instead of on the client side?